tasmota/language/it-IT.h

```diff
@@ -56,54 +56,54 @@
 #define D_AP "AP"                    // Access Point
 #define D_AS "come"
 #define D_AUTO "AUTO"
-#define D_BLINK "Blink"
-#define D_BLINKOFF "BlinkOff"
+#define D_BLINK "Lampeggia"
+#define D_BLINKOFF "Lampeggia OFF"
 #define D_BOOT_COUNT "Numero di boot"
 #define D_BRIGHTLIGHT "Luminoso"
 #define D_BSSID "BSSId"
 #define D_BUTTON "Pulsante"
 #define D_BY "da"                    // Written by me
-#define D_BYTES "Bytes"
+#define D_BYTES "Byte"
 #define D_CELSIUS "Celsius"
 #define D_CHANNEL "Canale"
 #define D_CO2 "CO2"
 #define D_CODE "codice"                // Button code
 #define D_COLDLIGHT "Fredda"
 #define D_COMMAND "Comando"
 #define D_CONNECTED "Connesso"
-#define D_CORS_DOMAIN "CORS Domain"
+#define D_CORS_DOMAIN "Dominio CORS"
 #define D_COUNT "Conteggio"
 #define D_COUNTER "Contatore"
-#define D_CT_POWER "CT Power"
+#define D_CT_POWER "Alimentazione CT"
 #define D_CURRENT "Corrente"          // As in Voltage and Current
 #define D_DATA "Dati"
 #define D_DARKLIGHT "Scuro"
 #define D_DEBUG "Debug"
-#define D_DEWPOINT "Dew point"
+#define D_DEWPOINT "Punto dew"
```

Comment:
Punto di rugiada.

Reply:
OK- Non è un DEV point (punto sviluppo)?

Reply:
No.
https://it.wikipedia.org/wiki/Punto_di_rugiada

Reply:
In questo caso sarebbe auspicabile l'aggiunta di un commento di fianco che indichi il contesto perché "Dew point" subito dopo Debug porta a pensare ad altre cose.
traduzioni
Nel caso di descrizione troppo corte o accorciate il commento a fianco che indichi il contesto è sempre utile (es. Current che può avere più significati...).

Reply:
Concordo. Aggiungilo pure.